<!doctype html>
<html lang="en">
<head>

  <meta charset="utf-8"><!-- Begin Jekyll SEO tag v2.3.0 -->
<title>Minimum Error Calibration and Normalization for Genomic Copy Number Analysis</title>
<meta property="og:title" content="Minimum Error Calibration and Normalization for Genomic Copy Number Analysis" />
<meta name="author" content="@mbaudis" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Minimum Error Calibration and Normalization for Genomic Copy Number Analysis. Bo Gao and Michael Baudis (2020) bioRxiv, 2019-07-31. DOI 10.1101/720854 Genomics, Volume 112, Issue 5, September 2020, Pages 3331-3341, accepted 2020-05-06 doi.org/10.1016/j.ygeno.2020.05.008." />
<meta property="og:description" content="Minimum Error Calibration and Normalization for Genomic Copy Number Analysis. Bo Gao and Michael Baudis (2020) bioRxiv, 2019-07-31. DOI 10.1101/720854 Genomics, Volume 112, Issue 5, September 2020, Pages 3331-3341, accepted 2020-05-06 doi.org/10.1016/j.ygeno.2020.05.008." />
<link rel="canonical" href="http://localhost:4000/standards/news/2020/05/06/publication-Mecan4CNA-Genomics.html" />
<meta property="og:url" content="http://localhost:4000/standards/news/2020/05/06/publication-Mecan4CNA-Genomics.html" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2020-05-06T00:00:00+02:00" />
<script type="application/ld+json">
{"name":null,"headline":"Minimum Error Calibration and Normalization for Genomic Copy Number Analysis","dateModified":"2020-05-06T00:00:00+02:00","datePublished":"2020-05-06T00:00:00+02:00","sameAs":null,"@type":"BlogPosting","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/standards/news/2020/05/06/publication-Mecan4CNA-Genomics.html"},"url":"http://localhost:4000/standards/news/2020/05/06/publication-Mecan4CNA-Genomics.html","description":"Minimum Error Calibration and Normalization for Genomic Copy Number Analysis. Bo Gao and Michael Baudis (2020) bioRxiv, 2019-07-31. DOI 10.1101/720854 Genomics, Volume 112, Issue 5, September 2020, Pages 3331-3341, accepted 2020-05-06 doi.org/10.1016/j.ygeno.2020.05.008.","author":{"@type":"Person","name":"@mbaudis"},"image":null,"publisher":null,"@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<meta name="viewport" content="width=device-width">
  <link rel="shortcut icon" href="/assets/img/site-specific/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" href="/assets/css/style.css">
	<script src="/assets/preload/clipboard.min.js"></script>
  <link rel="stylesheet" href="/assets/preload/site-style.css">
		</head>

<body>

<div id="headband"><img src="/assets/img/site-specific/site-logo-topleft.png" id="logo_topleft" /><img src="/assets/img/site-specific/site-logo-topright.png" id="logo_topright" /></div>

<div id="nav_r"> </div>

<div class="container" style="margin-top: 20px">

<header>

  <div id="sitelogo">
<a href="/">
		<h2>hCNV Community</h2>
  	<div style="font-size: 0.9em; margin-top: -10px; margin-bottom: 15px;">Genomic Copy Number Variations in Humans</div>
</a>
	</div>

  <div id="catlinks"><h4><a href="/categories/news.html">News & Events</a></h4><h4 class="sublink">
      <a href="/news/2022/06/07/h-CNV-ELIXIR-All-Hands.html">ELIXIR All Hands 2022 - h-CNV Representation</a>
    </h4><h4 class="sublink">
      <a href="/presentations/news/2022/03/15/Michael-Baudis__hCNV-implementation-studies__ELIXIR-Human-Data-Communities.html">hCNV Implementation Studies Old and New</a>
    </h4><h4 class="sublink">
      <a href="/news/2022/01/21/cnv-ontology-proposal-note.html">CNV Ontology Proposal - Now Live at EFO</a>
    </h4><h4 class="sublink">
      <a href="/news/2021/11/19/cnvar-domain.html">hCNV Site now at cnvar.org</a>
    </h4><h4 class="sublink" >
      <a href="/categories/news.html">all ...</a>
    </h4><h4><a href="/categories/people.html">Participants</a></h4><h4><a href="/categories/standards.html">Standards and Guidelines</a></h4><h4><a href="/categories/resources.html">Studies & Resources</a></h4><h4><a href="/categories/howto.html">Examples, Guides & FAQ</a></h4><h4><a href="/categories/contact.html">Contacts</a></h4></div><div id="outlinks"></div><div id="outlinks"><h5>Related Sites</h5><h4 class="sublink">
      <a href="http://genome.blog">Genome Blog</a>
    </h4><h4 class="sublink">
      <a href="https://elixir-europe.org/about-us/implementation-studies/2019-hcnv">h-CNV @ ELIXIR</a>
    </h4><h4 class="sublink">
      <a href="https://beacon-project.io">Beacon Project</a>
    </h4><h4 class="sublink">
      <a href="http://www.elixir-europe.org/about-us/implementation-studies/beacons">Beacon @ ELIXIR</a>
    </h4><h4 class="sublink">
      <a href="http://schemablocks.org">SchemaBlocks</a>
    </h4></div><div id="outlinks"><h5>Github Projects</h5><h4 class="sublink">
      <a href="http://github.com/hcnv/">h-CNV</a>
    </h4><h4 class="sublink">
      <a href="http://github.com/ga4gh-schemablocks/">SchemaBlocks</a>
    </h4><h4 class="sublink">
      <a href="http://github.com/ga4gh-beacon/">ELIXIR Beacon</a>
    </h4></div>

  <div id="tagcloud">

  <h5>Tags</h5><div class="site-tag">
      <a href="/tags/Beacon.html" style="font-size: 71%">Beacon</a>
    </div><div class="site-tag">
      <a href="/tags/FAQ.html" style="font-size: 74%">FAQ</a>
    </div><div class="site-tag">
      <a href="/tags/IS_2019-2021.html" style="font-size: 71%">IS 2019-2021</a>
    </div><div class="site-tag">
      <a href="/tags/IS_2021_Data.html" style="font-size: 77%">IS 2021 Data</a>
    </div><div class="site-tag">
      <a href="/tags/IS_2021_hCNV-X.html" style="font-size: 87%">IS 2021 hCNV-X</a>
    </div><div class="site-tag">
      <a href="/tags/code.html" style="font-size: 73%">code</a>
    </div><div class="site-tag">
      <a href="/tags/contacts.html" style="font-size: 73%">contacts</a>
    </div><div class="site-tag">
      <a href="/tags/databases.html" style="font-size: 73%">databases</a>
    </div><div class="site-tag">
      <a href="/tags/documentation.html" style="font-size: 76%">documentation</a>
    </div><div class="site-tag">
      <a href="/tags/hackathon.html" style="font-size: 72%">hackathon</a>
    </div><div class="site-tag">
      <a href="/tags/ontologies.html" style="font-size: 72%">ontologies</a>
    </div><div class="site-tag">
      <a href="/tags/presentation.html" style="font-size: 76%">presentation</a>
    </div><div class="site-tag">
      <a href="/tags/press.html" style="font-size: 72%">press</a>
    </div><div class="site-tag">
      <a href="/tags/publication.html" style="font-size: 74%">publication</a>
    </div><div class="site-tag">
      <a href="/tags/standards.html" style="font-size: 74%">standards</a>
    </div><div class="site-tag">
      <a href="/tags/website.html" style="font-size: 75%">website</a>
    </div></div>

</header><section>

<h3 id="minimum-error-calibration-and-normalization-for-genomic-copy-number-analysis">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis.</h3>
<h4 id="bo-gao-and-michael-baudis-2020">Bo Gao and Michael Baudis (2020)</h4>
<h5 id="biorxiv-2019-07-31-doi-101101720854">bioRxiv, 2019-07-31. <a href="https://doi.org/10.1101/720854">DOI 10.1101/720854</a></h5>
<h5 id="genomics-volume-112-issue-5-september-2020-pages-3331-3341-accepted-2020-05-06-doiorg101016jygeno202005008">Genomics, Volume 112, Issue 5, September 2020, Pages 3331-3341, accepted 2020-05-06 <a href="https://doi.org/10.1016/j.ygeno.2020.05.008">doi.org/10.1016/j.ygeno.2020.05.008</a>.</h5>
<!--more-->

<h4 id="abstract">Abstract</h4>

<h5 id="background">Background</h5>
<p>Copy number variations (CNV) are regional deviations from the normal autosomal bi-allelic DNA content. While germline CNVs are a major contributor to genomic syndromes and inherited diseases, the majority of cancers accumulate extensive “somatic” CNV (sCNV or CNA) during the process of oncogenetic transformation and progression. While specific sCNV have closely been associated with tumorigenesis, intriguingly many neoplasias exhibit recurrent sCNV patterns beyond the involvement of a few cancer driver genes. Currently, CNV profiles of tumor samples are generated using genomic micro-arrays or high-throughput DNA sequencing. Regardless of the underlying technology, genomic copy number data is derived from the relative assessment and integration of multiple signals, with the data generation process being prone to contamination from several sources. Estimated copy number values have no absolute or strictly linear correlation to their corresponding DNA levels, and the extent of deviation differs between sample profiles, which poses a great challenge for data integration and comparison in large scale genome analysis.</p>

<h5 id="results">Results</h5>
<p>In this study, we present a novel method named ”Minimum Error Calibration and Normalization for Copy Numbers Analysis” (Mecan4CNA). It only requires CNV segmentation files as input, is platform independent, and has a high performance with limited hardware requirements. For a given multi-sample copy number dataset, Mecan4CNA can batch-normalize all samples to the corresponding true copy num- ber levels of the main tumor clones. Experiments of Mecan4CNA on simulated data showed an overall accuracy of 93% and 91% in determining the normal level and single copy alteration (i.e. duplication or loss of one allele), respectively. Comparison of estimated normal levels and single copy alternations with existing methods and karyotyping data on the NCI-60 tumor cell line produced coherent results. To estimate the method’s impact on downstream analyses, we performed GISTIC analyses on the original and Mecan4CNA normalized data from the Cancer Genome Atlas (TCGA) where the normalized data showed prominent improvements of both sensitivity and specificity in detecting focal regions.</p>

<h5 id="conclusions">Conclusions</h5>
<p>Mecan4CNA provides an advanced method for CNA data normalization, especially in meta-analyses involving large profile numbers and heterogeneous source data quality. With its informative output and visualization options, Mecan4CNA also can improve the interpretation of individual CNA profiles. Mecan4CNA is freely available as a Python package and through its code repository on Github.</p>
<h4>Links</h4>

  <ul>
        <li><a href="https://www.biorxiv.org/content/10.1101/720854v1" target="_blank">[bioRxiv]</a></li>
    
        <li><a href="https://pypi.org/project/mecan4cna/" target="_blank">[Python pip]</a></li>
    
        <li><a href="https://github.com/baudisgroup/mecan4cna" target="_blank">[Github]</a></li>
    </ul><div class="pagestamp">2020-05-06
  </div></section>

<footer></footer>

</div></body>
</html>
