<!doctype html>
<html lang="en">
<head>

  <meta charset="utf-8"><!-- Begin Jekyll SEO tag v2.3.0 -->
<title>CNVariant</title>
<meta property="og:title" content="CNVariant" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="CNVariant [specifications] &nearr; {S}[B] Status Level [i] playground Provenance hCNV community project Used by ELIXIR hCNV community" />
<meta property="og:description" content="CNVariant [specifications] &nearr; {S}[B] Status Level [i] playground Provenance hCNV community project Used by ELIXIR hCNV community" />
<link rel="canonical" href="http://localhost:4000/schemas/specifications/CNVariant.html" />
<meta property="og:url" content="http://localhost:4000/schemas/specifications/CNVariant.html" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2020-05-29T00:00:00+02:00" />
<script type="application/ld+json">
{"name":null,"headline":"CNVariant","dateModified":"2020-05-29T00:00:00+02:00","datePublished":"2020-05-29T00:00:00+02:00","sameAs":null,"@type":"BlogPosting","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/schemas/specifications/CNVariant.html"},"url":"http://localhost:4000/schemas/specifications/CNVariant.html","description":"CNVariant [specifications] &nearr; {S}[B] Status Level [i] playground Provenance hCNV community project Used by ELIXIR hCNV community","author":null,"image":null,"publisher":null,"@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<meta name="viewport" content="width=device-width">
  <link rel="shortcut icon" href="/assets/img/site-specific/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" href="/assets/css/style.css">
	<script src="/assets/preload/clipboard.min.js"></script>
  <link rel="stylesheet" href="/assets/preload/site-style.css">
		</head>

<body>

<div id="headband"><img src="/assets/img/site-specific/site-logo-topleft.png" id="logo_topleft" /><img src="/assets/img/site-specific/site-logo-topright.png" id="logo_topright" /></div>

<div id="nav_r"> </div>

<div class="container" style="margin-top: 20px">

<header>

  <div id="sitelogo">
<a href="/">
		<h2>hCNV Community</h2>
  	<div style="font-size: 0.9em; margin-top: -10px; margin-bottom: 15px;">Genomic Copy Number Variations in Humans</div>
</a>
	</div>

  <div id="catlinks"><h4><a href="/categories/news.html">News & Events</a></h4><h4><a href="/categories/people.html">Participants</a></h4><h4><a href="/categories/standards.html">Standards and Guidelines</a></h4><h4><a href="/categories/resources.html">Studies & Resources</a></h4><h4><a href="/categories/howto.html">Examples, Guides & FAQ</a></h4><h4><a href="/categories/contact.html">Contacts</a></h4></div><div id="outlinks"></div><div id="outlinks"><h5>Related Sites</h5><h4 class="sublink">
      <a href="http://genome.blog">Genome Blog</a>
    </h4><h4 class="sublink">
      <a href="https://elixir-europe.org/about-us/implementation-studies/2019-hcnv">h-CNV @ ELIXIR</a>
    </h4><h4 class="sublink">
      <a href="https://beacon-project.io">Beacon Project</a>
    </h4><h4 class="sublink">
      <a href="http://www.elixir-europe.org/about-us/implementation-studies/beacons">Beacon @ ELIXIR</a>
    </h4><h4 class="sublink">
      <a href="http://schemablocks.org">SchemaBlocks</a>
    </h4></div><div id="outlinks"><h5>Github Projects</h5><h4 class="sublink">
      <a href="http://github.com/hcnv/">h-CNV</a>
    </h4><h4 class="sublink">
      <a href="http://github.com/ga4gh-schemablocks/">SchemaBlocks</a>
    </h4><h4 class="sublink">
      <a href="http://github.com/ga4gh-beacon/">ELIXIR Beacon</a>
    </h4></div>

  <div id="tagcloud">

  <h5>Tags</h5><div class="site-tag">
      <a href="/tags/Beacon.html" style="font-size: 71%">Beacon</a>
    </div><div class="site-tag">
      <a href="/tags/FAQ.html" style="font-size: 74%">FAQ</a>
    </div><div class="site-tag">
      <a href="/tags/IS_2019-2021.html" style="font-size: 71%">IS 2019-2021</a>
    </div><div class="site-tag">
      <a href="/tags/IS_2021_Data.html" style="font-size: 77%">IS 2021 Data</a>
    </div><div class="site-tag">
      <a href="/tags/IS_2021_hCNV-X.html" style="font-size: 87%">IS 2021 hCNV-X</a>
    </div><div class="site-tag">
      <a href="/tags/code.html" style="font-size: 73%">code</a>
    </div><div class="site-tag">
      <a href="/tags/contacts.html" style="font-size: 73%">contacts</a>
    </div><div class="site-tag">
      <a href="/tags/databases.html" style="font-size: 73%">databases</a>
    </div><div class="site-tag">
      <a href="/tags/documentation.html" style="font-size: 76%">documentation</a>
    </div><div class="site-tag">
      <a href="/tags/hackathon.html" style="font-size: 72%">hackathon</a>
    </div><div class="site-tag">
      <a href="/tags/ontologies.html" style="font-size: 72%">ontologies</a>
    </div><div class="site-tag">
      <a href="/tags/presentation.html" style="font-size: 76%">presentation</a>
    </div><div class="site-tag">
      <a href="/tags/press.html" style="font-size: 72%">press</a>
    </div><div class="site-tag">
      <a href="/tags/publication.html" style="font-size: 74%">publication</a>
    </div><div class="site-tag">
      <a href="/tags/standards.html" style="font-size: 74%">standards</a>
    </div><div class="site-tag">
      <a href="/tags/website.html" style="font-size: 75%">website</a>
    </div></div>

</header><section>

<div id="schema-header-title">
  <h2>CNVariant <span id="schema-header-title-project">[specifications] <a href="https://github.com/hcnv/specifications" target="_BLANK">&nearr;</a></span> </h2>
</div>

<table id="schema-header-table">
  <tr>
    <th>{S}[B] Status Level <a href="https://schemablocks.org/about/sb-status-levels.html">[i]</a></th>
    <td><div id="schema-header-status">playground</div></td>
  </tr>

  <tr>
    <th>Provenance</th>
    <td>
      <ul>
<li><a href="https://hcnv.github.io/schemas/specifications/">hCNV community project</a></li>
      </ul>
    </td>
  </tr>
  <tr>
    <th>Used by</th>
    <td>
      <ul>
<li>ELIXIR hCNV community</li>
      </ul>
    </td>
  </tr>

<!--more-->

  <tr>
    <th>Contributors</th>
    <td>
      <ul>
<li><a href="https://hcnv.github.io/categories/people.html">ELIXIR hCNV team</a></li>
<li><a href="https://orcid.org/0000-0002-9903-4248">Michael Baudis</a></li>
      </ul>
    </td>
  </tr>
  <tr>
    <th>Source (0.0.1-draft.1)</th>
    <td>
      <ul>
        <li><a href="current/CNVariant.json" target="_BLANK">raw source [JSON]</a></li>
        <li><a href="https://github.com/hcnv/specifications/blob/master/schemas/CNVariant.yaml" target="_BLANK">Github</a></li>
      </ul>
    </td>
  </tr>
</table>

<div id="schema-attributes-title">
  <h3>Attributes</h3>
</div>

<p><strong>Type:</strong> object<br />
<strong>Description:</strong> The document describes a draft variant schema, for modeling CN variant objects.</p>

<h3 id="properties">Properties</h3>

<table id="schema-properties-table">
  <tr>
    <th>Property</th>
    <th>Type</th>
  </tr>
  <tr>
    <th>basePloidy</th>
    <td>integer</td>
  </tr>
  <tr>
    <th>digest</th>
    <td>string</td>
  </tr>
  <tr>
    <th>featureAnnotations</th>
    <td>array of "object"</td>
  </tr>
  <tr>
    <th>id</th>
    <td>string</td>
  </tr>
  <tr>
    <th>info</th>
    <td>object</td>
  </tr>
  <tr>
    <th>referenceGenome</th>
    <td>string</td>
  </tr>
  <tr>
    <th>referenceId</th>
    <td>string</td>
  </tr>
  <tr>
    <th>svarEnd</th>
    <td>https://github.com/ga4gh/vr-spec/blob/116-patch-future-plans/schema/vr.json#118</td>
  </tr>
  <tr>
    <th>svarStart</th>
    <td>https://github.com/ga4gh/vr-spec/blob/116-patch-future-plans/schema/vr.json#118</td>
  </tr>
  <tr>
    <th>variantType</th>
    <td>object</td>
  </tr>

</table>

<h4 id="baseploidy">basePloidy</h4>

<ul>
  <li>type: integer</li>
</ul>

<p>Base ploidy of the sample for this reference. This may vary depending on organism or
specific cell/tissue type as well as for sex chromosomes. While CNVs are usually reported
as relative changes w/ respect to a “balanced” genome state, a <code class="language-plaintext highlighter-rouge">basePloidy</code> value
can help with the interpretation of absolute copy number counts.
Examples:</p>
<ul>
  <li>autosome in diploid karyotype: 2</li>
  <li>complete hydatiform mole: 1</li>
  <li>X-chromosome in karyotypically “normal” male: 1</li>
  <li>chromosome 2 in cell line NCI H-82 <code class="language-plaintext highlighter-rouge">cellosaurus:CVCL_1591</code>): 3</li>
</ul>

<h4 id="digest">digest</h4>

<ul>
  <li>
    <p>type: string</p>
  </li>
  <li>Concatenated unique specific elements of the variant.</li>
  <li>Optional, convenience element to derive unique variants in “individual
variant from callset” storage systems</li>
  <li>Possible use of e.g. HGVS-style annotations, but <em>not</em> defined as such</li>
</ul>

<h5 id="digest-value-example"><code class="language-plaintext highlighter-rouge">digest</code> Value Example</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>"4:12282-46465:DEL"
</code></pre></div></div>

<h4 id="featureannotations">featureAnnotations</h4>

<ul>
  <li>type: array of “object”</li>
</ul>

<h4 id="id">id</h4>

<ul>
  <li>
    <p>type: string</p>
  </li>
  <li>local-unique identifier of this variant (referenced as “variant_id”)</li>
  <li>Optional</li>
</ul>

<h5 id="id-value-example"><code class="language-plaintext highlighter-rouge">id</code> Value Example</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>"amvar-8754-7751-1119-8539"
</code></pre></div></div>

<h4 id="info">info</h4>

<ul>
  <li>type: object</li>
</ul>

<p>miscellaneous key-value pairs (where values can be objects)</p>

<h5 id="info-value-example"><code class="language-plaintext highlighter-rouge">info</code> Value Example</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>{
   "identifiers" : [
      "GPL6801"
   ],
   "platform" : "Affymetrix SNP6 Genotyping array",
   "platform_type" : "SNP array"
}
</code></pre></div></div>

<h4 id="referencegenome">referenceGenome</h4>

<ul>
  <li>type: string</li>
</ul>

<p>GRC reference genome edition</p>

<h5 id="referencegenome-value-example"><code class="language-plaintext highlighter-rouge">referenceGenome</code> Value Example</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>"GRCh38"
</code></pre></div></div>

<h4 id="referenceid">referenceId</h4>

<ul>
  <li>
    <p>type: string</p>
  </li>
  <li>Identifier for a reference sequence, generally describing a chromosome.</li>
  <li>The use of standard sequence identifiers with version (e.g. <code class="language-plaintext highlighter-rouge">NC_000009.12</code>)
obviates the separate need for a <code class="language-plaintext highlighter-rouge">referenceGenome</code> parameter’</li>
  <li>TODO: Stricter definition of identifier use</li>
</ul>

<h5 id="referenceid-value-examples"><code class="language-plaintext highlighter-rouge">referenceId</code> Value Examples</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>"NC_000009"
</code></pre></div></div>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>"NC_000006.12"
</code></pre></div></div>

<h4 id="svarend">svarEnd</h4>

<ul>
  <li>
    <p>type: https://github.com/ga4gh/vr-spec/blob/116-patch-future-plans/schema/vr.json#118</p>
  </li>
  <li>
    <p>Information about range in which the end of the CNV has been mapped. This
corresponds to the GA4GH GKS VRS “SimpleInterval”. Also see annotation for <code class="language-plaintext highlighter-rouge">svarStart</code>.</p>
  </li>
</ul>

<h5 id="svarend-value-example"><code class="language-plaintext highlighter-rouge">svarEnd</code> Value Example</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>{
   "end" : "138394717",
   "start" : "137000000"
}
</code></pre></div></div>

<h4 id="svarstart">svarStart</h4>

<ul>
  <li>
    <p>type: https://github.com/ga4gh/vr-spec/blob/116-patch-future-plans/schema/vr.json#118</p>
  </li>
  <li>Information about range in which the start of the CNV has been mapped. This
corresponds to the GA4GH GKS VRS “SimpleInterval”.</li>
  <li>Optional pointer to fusion partner? Position or ID?</li>
  <li>The positions are annotated using a 0-based “interbase” coordinate system
(<em>cave</em>: differs from HGVS and browser displays).</li>
</ul>

<h5 id="svarstart-value-example"><code class="language-plaintext highlighter-rouge">svarStart</code> Value Example</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>{
   "end" : "750050000",
   "start" : "75000000"
}
</code></pre></div></div>

<h4 id="varianttype">variantType</h4>

<ul>
  <li>type: object</li>
</ul>

<p>status as keyword or CURIE, quantitative parameters</p>

<h5 id="varianttype-value-example"><code class="language-plaintext highlighter-rouge">variantType</code> Value Example</h5>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>{
   "cn" : "3",
   "state" : "DUP"
}
</code></pre></div></div>
<hr />

<div id="schema-footer">
This schema representation is for information and testing purposes.
</div>

<div class="pagestamp">2020-05-29
  </div></section>

<footer></footer>

</div></body>
</html>
