---
title: "A systematic benchmark of copy number variation detection tools for high density SNP genotyping arrays"
template: post.html 
authors:
 - '@hangjiaz'
 - '@mbaudis'

date: 2024-11-14
pdf_file_name: 2024-11-14___Baardwijk-et-al.__benchmark-of-CNV-detection-tools-for-SNP-arrays__Genomics.pdf
links:
  - '[published at Genomics](https://www.sciencedirect.com/science/article/pii/S0888754324001836?via%3Dihub)'

---

#### M.N. van Baardwijk, L.S.E.M. Heijnen, H. Zhao, M. Baudis and A.P. Stubbs
##### Genomics (Elsevier). 2024 Nov 14.
* doi: [10.1016/j.ygeno.2024.110962](https://doi.org/10.1016/j.ygeno.2024.110962)
* PMID: [39547585](https://pubmed.ncbi.nlm.nih.gov/39547585/)


![elsevier genomics logo](/img/logo-elsevier-genomics.gif){ style="float: right; margin: -40px 0px 20px 20px;" }
**Abstract** Copy Number Variations (CNVs) are crucial in various diseases, especially cancer, but detecting them accurately from SNP genotyping arrays remains challenging. Therefore, this study benchmarked five CNV detection tools-PennCNV, QuantiSNP, iPattern, EnsembleCNV, and R-GADA-using SNP array and WGS data from 2002 individuals of the DRAGEN re-analysis of the 1000 Genomes project. Results showed significant variability in tool performance. R-GADA had the highest recall but low precision, while PennCNV was the most reliable in terms of precision and F1 score. EnsembleCNV improved recall by combining multiple callers but increased false positives.<!--more--> Overall, current tools, including new methods, do not outperform PennCNV in precise CNV detection. Improved reference data and consensus on true positive CNV calls are necessary. This study provides valuable insights and scalable workflows for researchers selecting CNV detection methods in future studies.
